assets/onboarding/features/feature-status.jsx

```diff
@@ -6,12 +6,12 @@ import CheckIcon from './check-icon';
 /**
  * Status constants.
  */
-export const LOADING_STATUS = 'loading';
+export const INSTALLING_STATUS = 'installing';
 export const ERROR_STATUS = 'error';
 export const INSTALLED_STATUS = 'installed';
 
 const statusComponents = {
-	[ LOADING_STATUS ]: (
+	[ INSTALLING_STATUS ]: (
 		<>
 			<Spinner />
 			<span className="screen-reader-text">
```

Comment:
The `Plugin installed` and `Error installing plugin` texts here should also be wrapped in `screen-reader-text` so that they are properly hidden. 
<img width="422" alt="image" src="https://user-images.githubusercontent.com/176949/83413042-74de0100-a41b-11ea-8f50-cabfa2b23291.png">

Reply:
I'll merge and check it in another PR ;)